libr/core/cmd_anal.c

```diff
@@ -9059,6 +9059,18 @@ static bool is_unknown_file(RCore *core) {
 	return true;
 }
 
+static bool is_apple_target(RCore *core) {
+	const char *arch = r_config_get (core->config, "asm.arch");
+	if (!strstr (arch, "ppc") && !strstr (arch, "arm") && !strstr (arch, "x86")) {
```

Comment:
shall this be `||`?

Reply:
No. translating this to V:

`if ["ppc", "arm", "ppc"].index(asm_arch) != -1 {`

objc on apple platforms is only avalble on ppc,arm and ppc. so if arch is not any of those it must return

Reply:
note its "strstr", not "strcmp"

Reply:
yeah sorry i misread the function indeed. LGTM